examples/src/heroes/aggregates/hero.aggregate.ts

```diff
@@ -1,17 +1,17 @@
+import { EventStoreAggregateRoot } from '../../../../src';
+import { HeroDamagedEnemyEvent } from '../events/impl/hero-damaged-enemy.event';
+import { HeroDropItemEvent } from '../events/impl/hero-drop-item.event';
 import { HeroFoundItemEvent } from '../events/impl/hero-found-item.event';
 import { HeroKilledDragonEvent } from '../events/impl/hero-killed-dragon.event';
-import { HeroDropItemEvent } from '../events/impl/hero-drop-item.event';
-import { HeroDamagedEnemyEvent } from '../events/impl/hero-damaged-enemy.event';
-import { EventStoreAggregateRoot } from '../../../../src';
 
 export class Hero extends EventStoreAggregateRoot {
-  constructor(private readonly id) {
+  constructor(private readonly id: string) {
```

Comment:
je suppose que le linter donne maintenant des warning quand on a pas les types de retours, ça serait peut etre inéressant de faire la même chose pour les accesseurs, non? (private, public, protected)

Reply:
Tu peux l'ajouter si tu veux oui

Reply:
ah oui mais si à chaque fois que j'ai une remarque sur tes PR tu me dis que je peux le faire, je comprends plus trop l'intérêt de passer par les PR ^^

Reply:
Parce que ça prend du temps et que je ne trouve pas ça indispensable ici. Dans le cadre d'une PR de maintenance l'idée c'était de rafraichir le package (donc le garder en l'état, updater les dépendances, fixer le lint et les tests).

Si on peut améliorer le package ? Oui. À plein de niveaux. Tu proposes une idée, je t'invite donc à la réaliser si elle te paraît suffisamment pertinente (: